iospi.yml

```diff
@@ -3,20 +3,23 @@ channels:
   - conda-forge
   - defaults
 dependencies:
+  #- codecov
```

Comment:
We should not have any leftover comments in the main codebase. If you want to keep them somewhere, you can put them on a separate branche.

Reply:
done